sarif-2.2/prose/edit/src/frontmatter.md

```diff
@@ -0,0 +1,105 @@
+<img src="media/OASISLogo-v3.0.png" style="width:3.01036in;height:0.61978in" />
+
+Static Analysis Results Interchange Format (SARIF) Version 2.1.0 Plus Errata 01 (redlined)
```

Comment:
We can remove the `(redlined)` notation.

Reply:
Will do, thanks.

Reply:
Now the frontmatter is aligned with the published errata at <https://docs.oasis-open.org/sarif/sarif/v2.1.0/errata01/os/sarif-v2.1.0-errata01-os-complete.html>